examples/multitenancy/appday2ops/web-app-chart/templates/post-install-job.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "{{ .Release.Name }}"
+  labels:
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+    helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+  annotations:
+    # This is what defines this resource as a hook. Without this line, the
+    # job is considered part of the release.
+    "helm.sh/hook": post-upgrade
+    "helm.sh/hook-weight": "-5"
+    "helm.sh/hook-delete-policy": hook-succeeded
+spec:
+  template:
+    metadata:
+      name: "{{ .Release.Name }}"
+      labels:
+        app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+        app.kubernetes.io/instance: {{ .Release.Name | quote }}
+        helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: post-install-job
+        image: mysql:5.6
+        command: ["mysql","--host=mysql","--user={{ .Values.db.username }}","--password={{ .Values.db.password }}","{{ .Values.db.db_name }}","-e",{{ .Values.sql.command | quote }}]
```

Comment:
host=mysql 

While this makes sense, we can make it configurable in values.yaml. Define a variable in values.yaml:
MYSQL_SERVICE_NAME: mysql

Modify the command to use "--host={{ .Values.MYSQL_SERVICE_NAME}}"

And modify mysql-deployment's service definition name to above as well.